tools/webhook-apicoverage/resourcetree/arraykindnode.go

```diff
@@ -28,38 +28,39 @@ const (
 
 // ArrayKindNode represents resource tree node of types reflect.Kind.Array and reflect.Kind.Slice
 type ArrayKindNode struct {
-	nodeData
+	NodeData
```

Comment:
This has to match the Type name, so cant be 'nodeData' and method is required because of anywhere outside the package, we call methods through NodeInterface, there is no access to this field.

Reply:
oh. I did not realize that it was the type and not the var.